services/common/src/components/common/CoreTableCommonColumns.tsx

```diff
@@ -50,11 +50,13 @@ export const renderCategoryColumn = (
   title: string,
   categoryMap: any,
   sortable = false,
-  placeHolder = EMPTY_FIELD
+  placeHolder = EMPTY_FIELD,
```

Comment:
Ugh I really want to kill this parameter (placeHolder)

Reply:
Okay onto review type things.  I'm happy enough to get rid of it, but I imagine there would be design implications there which should probably discussed before ditching it.